spec/basic_examples_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'hypothesis'
+
+def expect_failure(&block)
+  expect(&block).to raise_exception(RSpec::Expectations::ExpectationNotMetError)
+end
+
+RSpec.configure do |c|
```

Comment:
typically code like this would go in `spec/spec_helper.rb`. If you run `bundle exec rspec --init` it will generate a base `spec_helper.rb` file for you.